README.md

```diff
@@ -90,6 +90,7 @@ container.
         // Global variables passed to twig templates
         'ga_tracking' => 'UA-XXXXX-X'
     ],
+    'timezone' => 'default timezone identifier, e.g.: America/New_York'
```

Comment:
We have two ways we've handled this:

- As @timdev has done here, having descriptive text within the value, which then needs to be replaced.
- Alternately, having a default value, with descriptive text in an inline comment following the value.

I'm fine with either.